src/main/java/com/example/eventdrivennasa/service/NasaService.java

```diff
@@ -0,0 +1,29 @@
+package com.example.eventdrivennasa.service;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.stereotype.Service;
+
+import com.example.eventdrivennasa.model.LargestPicRequestDTO;
+
+import lombok.AllArgsConstructor;
+
+@Service
+@AllArgsConstructor
+public class NasaService {
+
+    private RabbitTemplate rabbitTemplate;
+
+    public String acceptAndPublish(LargestPicRequestDTO largestPicRequestDTO) {
+        var commandId = generateCommandId();
+        largestPicRequestDTO.setCommandId(commandId);
+        rabbitTemplate.convertAndSend("pictures-direct-exchange", "", largestPicRequestDTO);
```

Comment:
You use the property `pictures-direct-exchange` at least twice, I think it would be a good idea to extract it to a properties file and import it using the @Value annotation. This can be useful if you need to change this property, so you only need to change it in one place instead of searching for all usages in the project